### DISCUSSION TITLE
I recently started getting the "Browserslist: caniuse-lite is outdated" warning

### DISCUSSION THREAD

**Discussion by pkellner** [vercel/next.js#13135] [Help]
Don't know if it's NextJS related or not. I've searched around and don't see anything here on it and tried SO that suggested updating node (I did), removing package.json, I did.

Any suggestions? I get this warning exactly:

```
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
Browserslist: caniuse-lite is outdated. Please run next command `npm update`
```

**Comment by jamesmosier** [comment]
I believe this is just an upstream warning from a package that Next uses internally, likely `@babel/preset-env` or `postcss-preset-env`. 

![Screen Shot 2020-05-20 at 2 20 33 PM](https://user-images.githubusercontent.com/2854919/82482636-1fae0100-9aa5-11ea-8881-0e1e99def04a.png)
